src/Workspaces/Core/Portable/SymbolKey/SymbolKey.AliasSymbolKey.cs

```diff
@@ -78,7 +81,7 @@ public static SymbolKeyResolution Resolve(SymbolKeyReader reader, out string fai
                 {
                     if (child.IsNode)
                     {
-                        var result = Resolve(semanticModel, child.AsNode(), name, target, cancellationToken);
+                        var result = Resolve(semanticModel, child.AsNode()!, name, target, cancellationToken);
```

Comment:
Alas there's no way I can annotate IsNode to say AsNode will the return non-null, since it's a method. FYI to @jcouv if this is a common request.